### PR TITLE
Added support for standalone Twig loader outside of Drupal.

### PR DESCRIPTION
**Pull request by sun**
Problem
* The current Twig loader cannot be used outside of Drupal.

Goal
* Allow to use the Fractal integration including the special support for components without Drupal.

Proposed solution
1. Introduce a `TwigLoaderStandalone` that does not depend on Drupal or its services.
1. Add missing metadata so the package can be installed independently.

### CODE REVIEW

File: src/NodeVisitors/Attributes.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  protected $isAttribute = TRUE;
+
+  /**
+   * Checks if a node contains an attributes name expression.
+   *
+   * @return Twig_Node The modified node
+   */
+  public function doEnterNode(Twig_Node $node, Twig_Environment $env): object {
+    if ($node->hasAttribute('name') && $node->getAttribute('name') === 'attributes') {
```
Comment (sun) [inline src/NodeVisitors/Attributes.php]: Should this also cover all variables having "attributes" as suffix in their name as they are converted as well; e.g., `link_attributes`?

File: src/TwigLoaderStandalone.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      'atoms' => [$templateDir . '/components/1-atoms'],
+      'molecules' => [$templateDir . '/components/2-molecules'],
+      'organisms' => [$templateDir . '/components/3-organisms'],
+      'templates' => [$templateDir . '/components/4-templates'],
+      'pages' => [$templateDir . '/components/9-pages'],
+      'assets' => [$templateDir . '/dist'],
+    ];
+    foreach ($component_libraries as $namespace => $component_paths) {
+      foreach ($component_paths as $path) {
+        $this->prependPath($path, $namespace);
```
Comment (sun) [inline src/TwigLoaderStandalone.php]: This entire class should not be necessary as its only purpose is to support child themes extending parent themes.

However, the `getInstance()` method below had to be added as a temporary / quick hack to access the loader from the Component class, as it does not get the TwigEnvironment passed into it currently. Passing it as a dependency should enable access to the loader, and thus, making this class obsolete.

**Review by fabianmarz — COMMENTED** [review]
@sun, could you do a final review of my changes so we can get this merged? As it's working throughout the systems yet I don't think there are any critical issues to address.

**Pull request merged**